src/cars/car.service.ts

```diff
@@ -56,7 +57,7 @@ export class CarService {
      * @memberof CarService
      */
     create(carDto: CarDto, authUser: IAuthUser): Promise<InvokeResult> {
-        return this.requestHelper.invokeRequest(ChainMethod.createCar, carDto, authUser.id, false)
+        return this.requestHelper.invokeRequest(ChainMethod.createCar, carDto, EnvConfig.SKIP_MIDDLEWARE ? undefined : authUser.id, false)
```

Comment:
I would add this check in the invokeRequest function. Otherwise you would need to duplicate it every time.